src/components/Connect/Burner/CreateSeed/index.test.js

```diff
@@ -2,7 +2,7 @@ import { cleanup, render, screen, act, fireEvent } from '@testing-library/react'
 import composeMockAppTree from '../../../../test-utils/composeMockAppTree'
 import { flushPromises } from '../../../../test-utils'
 
-import Create from '.'
+import { CreateSeed } from '.'
```

Comment:
I don't like the default exports because it leaves room for the implementation to import it as whatever, creating discrepancies in component names